.github/workflows/tests.yml

```diff
@@ -322,16 +324,28 @@ jobs:
         if: ${{ github.event_name == 'schedule' }}
         run: echo "NOT_MARKERS=$NOT_MARKERS refcount" >> $GITHUB_ENV
 
+      - name: Export GitHub Actions cache environment variables
```

Comment:
I don't see how this is performing any caching. Can you explain? Why not using `actions/cache@v3`?

See [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows).

Reply:
this export env `ACTIONS_CACHE_URL` `ACTIONS_RUNTIME_TOKEN` which only exists when you call `uses:` but not `run:`, so we need to exprot these to env so vcpkg can use github actions cache.

Reply:
I see: vcpkg interacts actively with github action cache as a service, rather than having its artifacts cached.

Weird, but hey.